outputs/opentsdb/opentsdb.go

```diff
@@ -0,0 +1,151 @@
+package opentsdb
+
+import (
+	"fmt"
+	"net"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/influxdb/influxdb/client"
+    "github.com/influxdb/telegraf/outputs"
```

Comment:
please run `go fmt` on your code, it looks like you have some space-indented lines here